MIT.md

```diff
@@ -1,4 +1,4 @@
-Copyright 2023, Claudia Rojas | Pablo Bonasera | Alexandre Medina
```

Comment:
Agree again... It was hard and unexpected because even when we had troubles with their task I really wanted to help him. The new responsibilities forced me to work a lot, and you as well. Thank you for your support and hard work.